### PR TITLE
Add Chinese translation

### PR DESCRIPTION
**Pull request by hacamer**

### CODE REVIEW

**Review by hacamer — COMMENTED** [review]
fix Chinese translation mistake

File: 01.general/21.local-adguard/docs.zh.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+Windows，Mac和Android的AdGuard用户可能会注意到AdGuard在每个网页上添加了一个小脚本，该脚本是从`local.adguard.org`这个域名加载的。
+
+### 用于应用修饰规则的虚拟的域名
+
+首先，不用担心，这不是一个真实的域名，实际上并没有具有该名称的真实服务器。这个域名是用于对网页应用外观过滤，但一切都在您的设备上本地完成，无需连接到任何服务器。
+
+### 技术说明
+
+但是，这是怎么回事，为什么要这样做？请阅读以下技术说明
+1. Adguard是一个网络级内容阻止程序，因此它不能简单地将自定义JavaScript和CSS添加到网页中，例如浏览器扩展程序的操作。但是，这样做对于高质量的内容拦截至关重要。
```
Comment (DashaBu) [inline 01.general/21.local-adguard/docs.zh.md:18]: AdGuard 是一个网络级内容阻止程序，因此它不能简单地将自定义 JavaScript 和 CSS 添加到网页中，例如浏览器扩展程序的操作。

File: 01.general/21.local-adguard/docs.zh.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+2.为了做到这一点，adguard注入了一个看起来像这样的“内容脚本”：`<script src ="https://local.adguard.org/.../content-script.js">` 这个“内容脚本”会处理CSS过滤，隐藏或删除网页中的广告内容。
+3. `local.adguard.org`AdGuard 在网络级别拦截到域IP 地址的连接并在本地进行处理。这就是为什么该域具有多年不变的“静态”IP 地址。
```
Comment (DashaBu) [inline 01.general/21.local-adguard/docs.zh.md:20]: `local.adguard.org` AdGuard 在网络级别拦截到 IP 地址的连接并在本地进行处理。这就是为什么该域具有多年不变的“静态” IP 地址。

File: 01.general/21.local-adguard/docs.zh.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+**为什么我们需要使用真实的 IP 地址呢？**
+
+* 我们不能使用`127.0.0.1`，因为浏览器不接受。
+* 可以使用私有子网中的一些IP地址，但是该解决方案具有两个弊端
```
Comment (DashaBu) [inline 01.general/21.local-adguard/docs.zh.md:24]: 可以使用私有子网中的一些 IP 地址，但是该解决方案具有两个弊端。

File: 01.general/21.local-adguard/docs.zh.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    * 首先，很有可能与现有的Intranet服务相交并破坏对其的访问
```
Comment (DashaBu) [inline 01.general/21.local-adguard/docs.zh.md]: 首先，很有可能与现有的 Intranet 服务冲突并破坏对其的访问。

File: 01.general/21.local-adguard/docs.zh.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+    * 其次，一些DNS服务器可能会认为这是DNS重新攻击的攻击，并拒绝回应 ` local.adguard.org`
```
Comment (DashaBu) [inline 01.general/21.local-adguard/docs.zh.md]: 其次，一些 DNS 服务器可能会认为这是 DNS 重新绑定的攻击，并拒绝回应 ` local.adguard.org`。

**Review by DashaBu — CHANGES_REQUESTED** [review]

**Comment by el-termikael** [comment]
@hacamer Hi, thank you for adding Chinese translation! Please check the comments @DashaBu left to your pull request

**Comment by hacamer** [comment]
> @hacamer Hi, thank you for adding Chinese translation! Please check the comments @DashaBu left to your pull request

@el-termikael Hi ,I checked the comments and the modifications are correct but I don't know how to merge it🤣

**Review by el-termikael — APPROVED** [review]

**Comment by el-termikael** [comment]
@hacamer if you agree with the changes @DashaBu requested, you need to make them in your current branch and create one more commit
You can find more details in GitHub documentation: https://docs.github.com/en/pull-requests

**Comment by hacamer** [comment]
> @hacamer if you agree with the changes @DashaBu requested, you need to make them in your current branch and create one more commit You can find more details in GitHub documentation: https://docs.github.com/en/pull-requests

@el-termikael Hi I still don't know how to merge I have opened a new pull request this pull request will be closed
New pull request: #1118

**Pull request closed without merging**